src/catkin_pkg/workspaces.py

```diff
@@ -75,14 +75,21 @@ def order_paths(paths_to_order, prefix_paths):
     :param paths_to_order: list of paths
     :param prefix_paths: list of prefixes, must not end with '/'
     """
+    def contains(prefix, path):
+        prefix = os.path.realpath(prefix)
+        path = os.path.realpath(path)
```

Comment:
I would suggest to use os.path.normcase(os.path.realpath(...))` for these two lines.

That simplifies the following condition to: `path == prefix or path.startswith(prefix + os.sep)`.

Reply:
Spot on, done.